website/versioned_docs/version-V1/advanced/proxy/endpoints.mdx

```diff
@@ -1071,7 +1071,7 @@ Notifies the Proxy that the SDK with the given [SDK identifier](overview.mdx#sdk
 **Responses**:
 <ul className="responses">
 <li className="success"><span className="status">200</span>: The Proxy accepted the notification.</li>
-<li className="error"><span className="status">400</span>: The <code>sdkId</code> is missing or the <a href="/advanced/notifications-webhooks/#verifying-webhook-requests">webhook signature validation</a> failed.</li>
+<li className="error"><span className="status">400</span>: The <code>sdkId</code> is missing or the <a href="/docs/advanced/notifications-webhooks/#verifying-webhook-requests">webhook signature validation</a> failed.</li>
```

Comment:
It is a bit annoying now to handle both V1 and V2 pages, but on the V1 docs pages, we should stay on V1 pages after clicking on a link. 

```suggestion
<li className="error"><span className="status">400</span>: The <code>sdkId</code> is missing or the <a href="/docs/V1/advanced/notifications-webhooks/#verifying-webhook-requests">webhook signature validation</a> failed.</li>
```